Documentation/UserTsconfig/Setup.rst

```diff
@@ -159,4 +173,6 @@ titleLen
     positive integer
 
 :aspect:`Description`
-    Max. Title Length
+    Maximum length of rendered record titles in the BE interface.
+    It's used in several places: page tree, edit masks, workspace module, etc.
```

Comment:
```suggestion
    It is used in several places: page tree, edit masks, workspace module, etc.
```

We try to avoid `'`.
"edit masks" vs. "backend forms"? What do you think?